scripts/install-nix-from-closure.sh

```diff
@@ -83,18 +83,19 @@ while [ $# -gt 0 ]; do
 done
 
 if [ "$(uname -s)" = "Darwin" ]; then
-    if [ "$CREATE_DARWIN_VOLUME" = 1 ]; then
+    # TODO: not sure if it's worth also guarding this for pre-catalina?
+    if [ "$CREATE_DARWIN_VOLUME" = 1 ] && [ "$macos_major" -gt 14 ]; then
```

Comment:
I don't think we should be checking versions here, it's totally valid to use a volume on any system with apfs.

Reply:
Fair. 

My thinking is that we know it may cause some problems, so I don't want to encourage people who don't need it to do so?

But, I guess you could make the counter argument that if Catalina is going this way, it'd be better to just push all macOS users in this direction so that the install-base grows more uniform over time? 

In any case, I'll revert this one shortly.